organization/app/organization/[id]/turf-timeslot/page.tsx

```
@@ -0,0 +1,42 @@
+        <Suspense fallback={<TurfSelectionSkeleton />}>
+          <TurfSelection organizationId={organizationId} />
+        </Suspense>
+      </div>
+    </div>
+  );
+}
+
+function TurfSelectionSkeleton() {
+  return (
+    <div className="space-y-4">
+      <div className="h-12 bg-gray-200 rounded-md animate-pulse"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)